packages/lightcone_v2.js/src/wallet/privateKey.ts

```diff
@@ -142,12 +142,26 @@ export class PrivateKey {
    * @param gasPrice in gwei
    */
   public async depositTo(symbol: string, amount: number, gasPrice: number) {
```

Comment:
depositTo and withdrawFrom have a chain of API requests: exchange.deposit and this.account.sendTransaction
Using await and async is easier.

We can also use Promise here. However, we still need to catch errors.